packages/terra-open-source-scripts/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@cerner/terra-open-source-scripts",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Terra package with various scripts for open source development",
+  "main": "index.js",
+  "bin": {
+    "terra": "./bin/terra"
+  },
```

Comment:
I wouldn't have thought we needed to provide a bin if the cli is loading these commands.

Reply:
This was a copy paste err

Reply:
Fixed: https://github.com/cerner/terra-toolkit/pull/460/commits/6c35f4191a2115b9652c22dc9f42105554d0b08c